Sources/Signal.swift

```diff
@@ -1021,6 +1021,75 @@ extension SignalProtocol {
 			.map { $0.0 }
 	}
 
+	/// Forward the latest value from `samplee` with the value from `self` as a
+	/// tuple, only when `self` sends a `value` event.
+	/// This is like a flipped version of `sample(with:)` and Rx's `withLatestFrom`.
+	///
+	/// - note: If `self` fires before a value has been observed on `samplee`,
+	///         nothing happens.
+	///
+	/// - parameters:
+	///   - samplee: A signal that its latest value is sampled by `self`.
+	///
+	/// - returns: A signal that will send values from `self` and `samplee`,
+	///            sampled (possibly multiple times) by `self`, then terminate
+	///            once `self` has terminated. **`samplee`'s terminated events
+	///            are ignored**.
+	public func sample<U>(from samplee: Signal<U, NoError>) -> Signal<(Value, U), Error> {
```

Comment:
I prefer `withLatest(_:)` because "with" sounds like a small addition to the main stream, and receiver remains as the first class that also _implies_ the rank direction. 
Maybe `withLatest(from:)` can be a compromise, but I think we don't need this "from".